playbooks/files/grafana/ldap.toml.j2

```diff
@@ -0,0 +1,33 @@
+# {{ ansible_managed }}
+
+[[servers]]
+host = "{{ grafana_ldap__servers__host  }}"
+port = {{ grafana_ldap__servers__port  }}
+use_ssl = {{ grafana_ldap__servers__use_ssl  }}
+start_tls = {{ grafana_ldap__servers__start_tls  }}
+ssl_skip_verify = {{ grafana_ldap__servers__ssl_skip_verify  }}
+
+bind_dn = "{{ grafana_ldap__servers__bind_dn  }}"
+bind_password = '{{ grafana_ldap__servers__password  }}'
+
+search_filter = "{{ grafana_ldap__servers__search_filter  }}"
+search_base_dns = {{ grafana_ldap__servers__base_dns  }}
+
+[servers.attributes]
+name = "{{ grafana_ldap__servers_atributes__name  }}"
+surname = "{{ grafana_ldap__servers_atributes__surname  }}"
+username = "{{ grafana_ldap__servers_atributes__username  }}"
+member_of = "{{ grafana_ldap__servers_atributes__member_of  }}"
+email = "{{ grafana_ldap__servers_atributes__email  }}"
+
+[[servers.group_mappings]]
+group_dn = "{{ grafana_ldap__servers_group_mappings_1__group_dn  }}"
+org_role = "{{ grafana_ldap__servers_group_mappings_1__org_role  }}"
+
+[[servers.group_mappings]]
+group_dn = "{{ grafana_ldap__servers_group_mappings_2__group_dn  }}"
+org_role = "{{ grafana_ldap__servers_group_mappings_2__org_role  }}"
+
+[[servers.group_mappings]]
+group_dn = "{{ grafana_ldap__servers_group_mappings_3__group_dn  }}"
+org_role = "{{ grafana_ldap__servers_group_mappings_3__org_role  }}"
```

Comment:
If you change those variables to a single list of dicts, here you would do this:
```suggestion
{{
  dict(servers=dict(group_mappings=grafana_ldap__servers_group_mappings)) |
  tina_pm.common.to_toml
}}
```